packages/terra-form-select/src/shared/_Frame.module.scss

```diff
@@ -67,6 +67,22 @@
       border-color: var(--terra-form-select-focus-hover-border-color, #4cb2e9);
     }
 
+    &.is-incomplete:not(.is-invalid) {
+      background-color: #ffffcc; // var(--terra-form-select-incomplete-background-color, #ffffcc);
```

Comment:
Should this be themeable?

Reply:
Is now themeable https://github.com/cerner/terra-core/commit/93158da0ccc2be1d72588890924879e913215a96#diff-e1b4f6af96982d97ca070e391b465fc8R71